### ISSUE TITLE
Seems to be broken on windows 11

### ISSUE THREAD

**Issue by Ethanscharlie** [Mr-Technician/BorderlessMinecraft#24]
![image](https://user-images.githubusercontent.com/96396138/146693624-3ede9981-c934-4d67-ac92-29cb5da608d8.png)

**Comment by roshank231** [comment]
I'm running it on windows 11 just fine.

**Comment by Mr-Technician** [comment]
It's probably a permissions issue or something to do with .NET Framework. Can both of you provide more details about the windows builds you're running?

It could also be a code signing issue.

@roshank231 @Ethanscharlie

**Comment by Mr-Technician** [comment]
And version 1.3.0 of Borderless Minecraft?

**Comment by roshank231** [comment]
Yeah, 1.30 latest of boardless too.

**Comment by Asznee1** [comment]
I'm trying to use it in windows 10 but its not working, same message

**Comment by Mr-Technician** [comment]
> I'm trying to use it in windows 10 but its not working, same message

Latest 1.3.1?

**Comment by Asznee1** [comment]
yes, the latest version. Its weird because it says its 54KB in the download page but when I go to my downloads it says it's 0KB

**Comment by Mr-Technician** [comment]
I wonder if the download didn't work correctly. Could you try downloading the latest version again?

**Comment by Asznee1** [comment]
> I wonder if the download didn't work correctly. Could you try downloading the latest version again?

I tried downloading it multiple times and none of them work

**Comment by Mr-Technician** [comment]
@Asznee1 Can you download the latest again, go to your downloads folder, and follow these steps?

1. Hold shift and right click
2. click "Open Powershell window here"
3. Paste and run this command `Get-FileHash BorderlessMinecraft.exe`

The result should look like this:

The result should be a hash code like this: `CF77A32CFAA9FC53DC6A37A81E124C2B4740E303048756BD5D0C24AFFAF98141`

**Comment by Asznee1** [comment]
> 

After following those steps I get this Hash E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855
What do I do after?

**Comment by Mr-Technician** [comment]
> After following those steps I get this Hash E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855 What do I do after?

Make sure you ran the hash against the latest version (1.3.1) because that value should match my hash.
